src/modules/getScores.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+import API_SCORES_URL from './consts';
+import displayScores from './displayScores';
+
+const getAllScores = async () => {
+  try {
+    const response = await axios.get(API_SCORES_URL);
```

Comment:
- [OPTIONAL] I think 🤔  you use `async` and `await` with the `fetch` method to send and receive data for this project. but actually, you use Axios

Reply:
Okay. The project requirement demanded we get data from the API without being specific on how to get it. I will take a second look at it and if needed, I will change from Axios to fetch. Thanks.